packages/icons/src/leg-icon.js

```diff
@@ -20,7 +20,7 @@ const LegIcon = ({ getCompanyIcon, leg, ModeIcon, ...props }) => {
   // Do this for P&R, K&R and TNC trips without company icon
   if (iconStr && iconStr.startsWith("CAR")) iconStr = "CAR";
 
-  return <ModeIcon mode={iconStr} {...props} />;
+  return <ModeIcon mode={iconStr} leg={leg} {...props} />;
```

Comment:
`leg` is on props so doesn't this get passed anyway?

Reply:
I would have thought so too but it's not getting passed. I think because `getCompanyIcon`, `leg`, and `ModeIcon` all get destructured and so `props` winds up being what's left over? I could be wrong about why

Reply:
Oooh that's a good point. Honestly this is probably the right way to do it anyway...